### PR TITLE
Feature/#215 experiments data cms

### PR DESCRIPTION
**Pull request by will0684**
# Description

[Pull data from Strapi to display experiments](https://trello.com/c/BAQmDqRY/215-215-pull-data-from-strapi-to-display-experiments)

This task covers using getStaticProps in the Experiments page to fetch available experiments from the CMS api and use that data to render experiments on the page. Working with translation.

## Acceptance Criteria

Experiments shown in the Experiments page should be fetched from the CMS api and should be fully translated.

## Test Instructions

1. npm run dev
2. navigate to Experiments page
3. see experiments shown in the grid
4. view the fetch under Network in broswer devtools

## Help Requested

- Anything you think could be improved or that I missed

## Checklist

- [x] Strings use placeholders for translation (No hard coded strings)

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: pages/experiments.js [hunk 1]
```diff
@@ -27,23 +27,34 @@ export default function experiments(props) {
   const displayExperiments = filteredExperiments.map((experiment) => (
     <li key={experiment.id} className="flex items-stretch">
       <Experiment
-        title={experiment.title}
-        tag={experiment.tag}
-        tagLabel={t(experiment.tag)}
-        description={experiment.description}
+        title={
+          props.locale === "en"
```
Comment (Moro-Code) [inline pages/experiments.js:31]: I would say make en the default probably

**Review by Moro-Code — COMMENTED** [review]
code LGTM. Minor comments if you can address. Will approve when tests pass so I don't need to keep re approving

File: pages/experiments.js (hunk 1 above)
Comment (will0684) [reply pages/experiments.js:31]: Nvm I think I see what you mean

**Review by EphraimAssi — APPROVED** [review]
It's working really well!

**Review by EphraimAssi — DISMISSED** [review]

**Review by pcraig3 — COMMENTED** [review]
Looks good! I just asked for 1 change (adding the new vars to the example env file).

File: .github/workflows/test-and-lint.yml [hunk 2]
```diff
@@ -17,6 +17,9 @@ jobs:
 
       - name: Build 🔧
         run: npm run build
+        env:
+          STRAPI_API_BACKEND_URL: https://alphasite-api.dts-stn.com
```
[the remark this answers is not in the document]
Comment (will0684) [reply .github/workflows/test-and-lint.yml:21]: Personally I think it's ok. I can't think of a reason to hide what will essentially be a public API anyways. 🤷‍♂️ Of course, I could be wrong in my understanding though.

File: README.md [hunk 3]
```diff
@@ -42,6 +42,10 @@ Here is a list of all the environment variables used to configure the applicatio
 
 `NOTIFY_BASE_API_URL`: The base url for the Notify api. This value should be https://api.notification.canada.ca
 
+`NEXT_PUBLIC_API_MOCKING`: Value can be `enabled` or `disabled`. If enabled, a mock service worker will run in the Node environment to intercept handled network requests in NextJS data fetching methods like getStaticProps(). We use this to mock API responses during Cypress testing.
+
+`STRAPI_API_BACKEND_URL`: Root URL for the Strapi CMS API. We use https://alphasite-api.dts-stn.com/ for our application.
+
```
[the remark this answers is not in the document]
Comment (will0684) [reply README.md:48]: Added!

**Review by EphraimAssi — APPROVED** [review]

**Review by Moro-Code — APPROVED** [review]
Jordan. This is very nice clean work. Super well done. Have we modified teamcity with the extra variable ?

**Comment by will0684** [comment]
> Jordan. This is very nice clean work. Super well done. Have we modified teamcity with the extra variable ?

Yup it's been added to the build pipeline 👍

**Pull request merged**